drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -2846,6 +2849,21 @@ VkRenderPass RenderingDeviceVulkan::_render_pass_create(const Vector<AttachmentF
 					description.initialLayout = VK_IMAGE_LAYOUT_UNDEFINED; //don't care what is there
 				}
 			} break;
+			case INITIAL_ACTION_DROP: {
```

Comment:
This was missing btw, if you want to copy an entire rect to a framebuffer and dont care for the contents, this is better than keep.